src/Component/Symbolizer/LineEditor/LineEditor.tsx

```diff
@@ -167,18 +169,44 @@ export class LineEditor extends React.Component<LineEditorProps> {
     }
   }
 
+  handleComposition = (composition: Compositions, field:string, onChange: Function, fallback: React.ReactElement) => {
```

Comment:
I would be prefer `defaultElement` over `fallback`.

Reply:
solved in https://github.com/terrestris/geostyler/pull/1186/commits/4fcbbd578e14eb8c7466107a521dd20ed8a860f5